packages/select/src/Select/Select.tsx

```diff
@@ -574,8 +574,8 @@ export const Select = forwardRef<HTMLDivElement, SelectProps>(
 
               {description && (
                 <Description
-                  data-lgid={lgIds.root}
-                  data-testid={lgIds.root}
+                  data-lgid={lgIds.description}
```

Comment:
If we make this change, then we should update the test utils. Currently, in a few test utils, we are calling the typography getLgIds function with the root, but with this change, we'll no longer need to do that. 

https://github.com/mongodb/leafygreen-ui/blob/b9fb557268ed34f8d2f06563ce69117203de2d45/packages/select/src/testing/utils/getTestUtils.ts#L22